ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.6.yaml

```diff
@@ -1,12 +1,10 @@
-base_images:
-  os:
-    name: ubi
-    namespace: ocp
-    tag: "7"
-binary_build_commands: make build
 build_root:
   project_image:
-    dockerfile_path: openshift-ci/Dockerfile.tools
+    dockerfile_literal: |
+      FROM registry.access.redhat.com/ubi8/ubi
+      RUN yum install -y --nodocs jq python3-virtualenv make git
+      RUN curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+      RUN chmod +x /usr/bin/kubectl
```

Comment:
and do we really need to download kubectl when line 22 specify it?

Reply:
and we cannot create a symlink at this point because the `oc` is added in later

Reply:
that works